### ISSUE TITLE
Please update your app following the new "Background Execution Limits" before targeting SDK 26.

### ISSUE THREAD

**Issue by Foxsterc** [OneSignal/OneSignal-Xamarin-SDK#54]
I'm guessing this is because Android O is not supported in this SDK?
```Java
java.lang.RuntimeException: Unable to start receiver com.onesignal.BootUpReceiver: java.lang.IllegalStateException: Not allowed to start service Intent { cmp=com.moonclimbing.moonboard/com.onesignal.NotificationRestoreService (has extras) }: app is in background uid UidRecord{78cf79e u0a156 RCVR idle change:uncached procs:1 seq(0,0,0)}
  at android.app.ActivityThread.handleReceiver(ActivityThread.java:3194)
  at android.app.ActivityThread.-wrap17(Unknown Source:0)
  at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1672)
  at android.os.Handler.dispatchMessage(Handler.java:106)
  at android.os.Looper.loop(Looper.java:164)
  at android.app.ActivityThread.main(ActivityThread.java:6494)
  at java.lang.reflect.Method.invoke(Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
Caused by: java.lang.IllegalStateException: Not allowed to start service Intent { cmp=com.moonclimbing.moonboard/com.onesignal.NotificationRestoreService (has extras) }: app is in background uid UidRecord{78cf79e u0a156 RCVR idle change:uncached procs:1 seq(0,0,0)}
  at android.app.ContextImpl.startServiceCommon(ContextImpl.java:1521)
  at android.app.ContextImpl.startService(ContextImpl.java:1477)
  at android.content.ContextWrapper.startService(ContextWrapper.java:650)
  at android.content.ContextWrapper.startService(ContextWrapper.java:650)
  at android.support.v4.content.WakefulBroadcastReceiver.startWakefulService(WakefulBroadcastReceiver.java:99)
  at com.onesignal.BootUpReceiver.onReceive(BootUpReceiver.java:42)
  at android.app.ActivityThread.handleReceiver(ActivityThread.java:3187)
  ... 8 more
```

**Comment by Foxsterc** [comment]
The silence is deafening...

**Comment by jkasten2** [comment]
@Foxsterc The OneSignal Xamarin SDK hasn't been updated to the latest OneSignal Android SDK to support targeting 26+ yet. You will need to lower this under Project Options > Build > Android Application. Then change the "Target Android version" to API 25.

**Comment by SeeD-Seifer** [comment]
Would be great to get update Xamarin SDK.

**Comment by alexshikov** [comment]
@jkasten2 any plans to update to the latest Android SDK?

**Comment by DragosBarbu** [comment]
Any news on this?

**Comment by Im-PJ** [comment]
Yes this xamarin sdk needs an update. Its currently not supporting .Net standard and API 26. This is really

**Comment by ndastur** [comment]
Judging by the fact that simple updates to fix error reporting have been outstanding for a year the chance of any updates is less than seeing a pig fly

**Comment by jkasten2** [comment]
Apologize for the lack updates, we'll be releasing an update this coming month to address this issue and others. Created issue #62 with more details. Thanks

**Comment by jkasten2** [comment]
This issue is addressed in the 3.1.0-alpha update if you want to give it a try to confirm.
https://github.com/OneSignal/OneSignal-Xamarin-SDK/releases
